net/lotus/module_test.go

```diff
@@ -0,0 +1,89 @@
+package lotus
+
+import (
+	"context"
+	"testing"
+
+	"github.com/textileio/powergate/net"
+	"github.com/textileio/powergate/tests"
+)
+
+var (
+	ctx = context.Background()
+)
+
+func TestModule(t *testing.T) {
```

Comment:
It's cool all these methods actually work in the dev net. It sees the miner as peer.